pallets/gear/src/tests.rs

```diff
@@ -15856,25 +15890,38 @@ mod utils {
 
     #[track_caller]
     pub(super) fn assert_responses_to_user(user_id: AccountId, assertions: Vec<Assertion>) {
```

Comment:
21 lines into 34 lines, no idea why to apply this refactor. readability for me just the same.

Reply:
```rust
if messages.len() != assertions.len() {
```

Before these lines if assertions amount is less than amount of events then you get `out of bounds` panic and no info about already converted events